docker-compose.yml

```diff
@@ -1019,6 +1023,38 @@ services:
         /arrow/ci/scripts/python_build.sh /arrow /build &&
         /arrow/ci/scripts/r_test.sh /arrow"
 
+  ubuntu-r-static:
```

Comment:
`ubuntu-r-static` ideally should be removed and called with:

```bash
archery docker run -e ARROW_BUILD_STATIC=ON -e ANOTHER_ENV_VAR=ETC  ubuntu-r
```

Reply:
See an [example of a static C++ build](https://github.com/apache/arrow/blob/master/dev/tasks/tasks.yml#L802-L809) using the `ubuntu-cpp` image.

Reply:
Ah, ha! That's what I was missing. Thanks for that pointer